src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -136,7 +149,11 @@ extends AbstractMap[Key, Value]
       None
     } else {
       val res = entry.value
```

Comment:
It's a no-op only if key carries no extra information other than that implied by equality.

But, to rephrase what I said previously: I argue that in `remove`, the `key` should be nulled out, to avoid memory leaks in cases where `Key` is some expensive object (which is not the goal of this PR, but should probably be addressed separately).

I'm not aware if this was discussed previously on the mailing list.